nft_ingester/src/bin/ingester/main.rs

```diff
@@ -694,15 +709,34 @@ pub async fn main() -> Result<(), IngesterError> {
         Ok(())
     });
 
-    if config.run_sequence_consistent_checker {
-        let force_reingestable_slot_processor = Arc::new(ForceReingestableSlotGetter::new(
+    let force_reingestable_slot_processor = Arc::new(ForceReingestableSlotGetter::new(
+        primary_rocks_storage.clone(),
+        Arc::new(DirectBlockParser::new(
+            tx_ingester.clone(),
             primary_rocks_storage.clone(),
-            Arc::new(DirectBlockParser::new(
-                tx_ingester.clone(),
-                primary_rocks_storage.clone(),
-                metrics_state.backfiller_metrics.clone(),
-            )),
-        ));
+            metrics_state.backfiller_metrics.clone(),
+        )),
+    ));
+
+    consistency_calculator::run_bg_consistency_calculator(
+        nft_change_rcv,
+        primary_rocks_storage.clone(),
+        force_reingestable_slot_processor.clone(),
+        shutdown_rx.resubscribe(),
+        metrics_state.checksum_calculation_metrics.clone(),
+    );
+
+    if let Some(peer_urls_file) = config.peer_urls_file.as_ref() {
```

Comment:
Do we need to add mounting for this file in docker compose?

Reply:
Probably, we do...